lib/Ama/Model/Votes.pm

```diff
@@ -3,12 +3,13 @@ use Mojo::Base -base;
 
 has 'pg';
 has 'username';
+has 'vote_floor';
 
 sub cast {
   my ($self, $entry_type, $entry_id, $vote) = @_;
+  my $sql;
```

Comment:
I don't think it makes sense for $sql to be declared outside of the only block it's needed in (L11)